### PR TITLE
Prompt for confirmation for unlisted full review (bug 1186369)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1186369

request-full-review-unlisted mov

### CODE REVIEW

File: apps/devhub/templates/devhub/versions/list.html [hunk 1]
```diff
@@ -73,7 +73,11 @@
                       amo.STATUS_LITE: _('Request Preliminary Review')} %}
         {% for status in request_reviews %}
           <form method="post"
-                action="{{ url('devhub.request-review', addon.slug, status) }}">
+                action="{{ url('devhub.request-review', addon.slug, status) }}"
+                {% if not addon.is_listed and status == amo.STATUS_PUBLIC %}
+                id="form-full-review"
```
Comment (magopian) [inline apps/devhub/templates/devhub/versions/list.html]: Is this needed? If so, maybe move it out of the `{% if ... %}` block?

File: static/js/zamboni/devhub.js [hunk 2]
```diff
@@ -188,6 +188,26 @@ $(document).ready(function() {
             $modalUnlist.render();
         }
     }
+    $(document.body).delegate('[data-confirm]', 'submit', function(e) {
```
Comment (magopian) [inline static/js/zamboni/devhub.js]: Could you please add a comment here, explaining what this does? Eg "display the confirmation modal on form submission."

**Comment by magopian** [comment]
Nice!

r+wc

File: apps/devhub/templates/devhub/versions/list.html (hunk 1 above)
Comment (mstriemer) [inline apps/devhub/templates/devhub/versions/list.html]: Looks like I removed the code that used it. I'll get rid of it.

**Pull request merged**